package.json

```diff
@@ -45,10 +45,12 @@
     "webpack-dev-server": "^2.4.4"
   },
   "dependencies": {
+    "bootstrap": "^3.3.7",
     "config": "^1.25.1",
     "crypto": "0.0.3",
     "express": "^4.15.2",
     "node-uuid": "^1.4.8",
+    "react-bootstrap": "^0.30.10",
```

Comment:
move this and bootstrap to dev-dep. For frontend dependencies use `npm install packageName --save-dev`